src/sprout/Controllers/Admin/ManagedAdminController.php

```diff
@@ -63,23 +65,47 @@
 **/
 abstract class ManagedAdminController extends Controller {
     /**
-    * This is the name of the controller - should match the class name, but without the '_Controller' bit.
-    **/
+     * This is the shorthand name of the controller.
+     *
+     * DO NOT declare this in the extending class. This is derived from the
```

Comment:
We should add a migration note for this, or maybe look for it in the module migration tool